.gitignore

```diff
@@ -1,29 +1,4 @@
-# Compiled Object files, Static and Dynamic libs (Shared Objects)
```

Comment:
No, I didn't know you were partial to it, looked like a default generated gitignore from github.